assets/js/app/LocusZoom.js

```diff
@@ -31,15 +31,15 @@ LocusZoom.addInstanceToDivById = function(id, datasource, layout, state){
         inst.mapTo(+region[0], +region[1], +region[2]);
     }
     return inst;
-}
+};
     
 // Automatically detect divs by class and populate them with default LocusZoom instances
 LocusZoom.populate = function(selector, datasource, layout, state) {
     if (typeof selector === "undefined"){
-        selector = ".lz-instance";
+        throw ("LocusZoom.populate selector not defined");
     }
     if (typeof layout === "undefined"){
-        layout = JSON.parse(JSON.stringify(LocusZoom.DefaultLayout));
+        layout = {};
```

Comment:
Latest commit removes those sanity checks from populate, since the Instance constructor is doing them in a more complete way (merging with defaults and whatnot). Arguably that's where that logic belongs, so now populate is a thinner pass-through and will silently pass and undefined parameters to the Instance constructor to gracefully handle.